language/java/run-tests.md

```diff
@@ -16,7 +16,7 @@ Testing is an essential part of modern software development. Testing can mean a
 
 ## Refactor Dockerfile to run tests
 
-The **Spring Pet Clinic** source code has already tests defined in the test directory `src/test/java/org/springframework/samples/petclinic`. You just need to update the JaCoCo version in your `pom.xml` to ensure your tests work with JDK v15 or higher with `<jacoco.version>0.8.6</jacoco.version>`, so we can use the following Docker command to start the container and run tests:
+The **Spring Pet Clinic** source code has already tests defined in the test directory `src/test/java/org/springframework/samples/petclinic`. We can use the following Docker command to start the container and run tests:
```

Comment:
Perhaps we already use it extensively on this page, but I usually try to avoid "We", so would use, e.g.;

```suggestion
The **Spring Pet Clinic** source code has already tests defined in the test directory `src/test/java/org/springframework/samples/petclinic`. Use the following command to start the container and run tests:
```